march_hardware/CMakeLists.txt

```diff
@@ -79,16 +79,10 @@ add_library(march_hardware
         src/MarchRobot.cpp
         src/TemperatureGES.cpp
         src/PDOmap.cpp
-        src/main.cpp
```

Comment:
Why are these changes in the Cmakelist needed?

Reply:
Because I removed the old 'main.cpp' file. It was never used and all it does at the moment is crash on refactors :)